linters/translation_filter_scanner.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+const PIPE_T_RXP = /\{\{\s*('|")((?:(?!\1).)+)\1\s*\|\s*t(?:ranslate)?(?::(.*?))?\s*(?:\||\}\})/g;
```

Comment:
Here's the diff in a more readable format

<img width="817" alt="image" src="https://user-images.githubusercontent.com/325821/72489320-b14cef80-37e1-11ea-9375-4b7f27974ed1.png">